chain/chain/src/store_validator.rs

```diff
@@ -39,7 +40,7 @@ pub struct StoreValidatorCache {
     tail: BlockHeight,
     chunk_tail: BlockHeight,
     block_heights_less_tail: Vec<CryptoHash>,
-    gc_col: Vec<u64>,
+    gc_col: enum_map::EnumMap<DBCol, u64>,
```

Comment:
Could you add a comment here for the field? Maybe we can rename it to `gc_count`. I had to read the code to figure out how this field is used.